srcpkgs/nsnake/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'nsnake'
+pkgname=nsnake
+version=3.0.1
+revision=1
+wrksrc=nSnake-${version}
```

Comment:
because you can't create a new separate thread to comment on the same line of code